docs/sftt/index.md

```diff
@@ -2,4 +2,6 @@
 
 [API Spec](./sftt-api-spec.md)
 
-[Auth API](./sftt-auth-api.md)
\ No newline at end of file
+[Auth API](./sftt-auth-api.md)
+
+[Map Frontend](./sfft-map-frontend.md)
```

Comment:
I'm confused, the file you edited is `docs/sftt/map-architecture.md`? 

In any case let's keep all map docs in one place